src/main/java/de/metas/ui/web/product/process/WEBUI_M_Product_LotNumber_Lock.java

```diff
@@ -50,28 +50,54 @@
  */
 
 /**
- * https://github.com/metasfresh/metasfresh/issues/3693
- * This process will search for HUs containing products with LotNo attributes that fit the pairs in the selected I_M_Product_LotNumber_Lock entries.
- * If such HUs are found, they will all be put in a DD_Order and sent to the Quarantine warehouse.
+ * https://github.com/metasfresh/metasfresh/issues/3693 This process will search
+ * for HUs containing products with LotNo attributes that fit the pairs in the
+ * selected I_M_Product_LotNumber_Lock entries. If such HUs are found, they will
+ * all be put in a DD_Order and sent to the Quarantine warehouse.
  * 
  * @author metas-dev <dev@metasfresh.com>
  *
  */
-public class WEBUI_M_Product_LotNumber_Lock extends ViewBasedProcessTemplate implements IProcessPrecondition
+public class WEBUI_M_Product_LotNumber_Lock extends ViewBasedProcessTemplate
+		implements
+		IProcessPrecondition
 {
-	private final IInvoiceCandBL invoiceCandBL = Services.get(IInvoiceCandBL.class);
+	private final IInvoiceCandBL invoiceCandBL = Services
+			.get(IInvoiceCandBL.class);
 	private final IHUInOutDAO huInOutDAO = Services.get(IHUInOutDAO.class);
 
+	private List<HUToDistribute> husToDistribute = new ArrayList<>();
```

Comment:
please align this variable name wiht the method(s) that use it

Reply:
Renamed it to hussToQuarantine